README.md

```diff
@@ -1563,6 +1562,13 @@ The example shows how to use custom metrics API in Python Backend. You can find
 the complete example instructions in
 [examples/custom_metrics](examples/custom_metrics/README.md).
 
+## PyTorch models
+
+The example shows how to use the
+[PyTorch platform handler](src/resources/platform_handlers/pytorch/README.md) to
```

Comment:
```suggestion
[PyTorch platform](src/resources/platform_handlers/pytorch/README.md) to
```

I think the README should move to a new docs folder or just be embedded in the main `README.md` under the `PyTorch models` section.

Reply:
> I think the README should move to a new docs folder or just be embedded in the main README.md under the PyTorch models section.

This is a good point. It makes PyTorch platform more findable.

Reply:
Merged PyTorch docs into main Python docs.